setup.py

```diff
@@ -90,6 +90,7 @@ def get_version():
         "cloudpickle >= 1.2.0",
         "importlib_metadata >= 4.8.0; python_version < '3.10'",
         "gymnasium_notices >= 0.0.1",
+        "shimmy>=0.1.0, <1.0",
```

Comment:
How confident are we that e.g. `shimmy==0.2` will not break anything when it happens?